package.json

```diff
@@ -85,7 +85,5 @@
     "webpack": "^5.74.0"
   },
   "dependencies": {},
-  "resolutions": {
-    "libnpmpublish": "4.0.0"
```

Comment:
I tried to understand the reason of this resolution, but could not found any reference. I think it's unnecessary now.